### ISSUE TITLE
Request: Slideshow gewünscht!!

### ISSUE THREAD

**Issue by ChrisWoehrle** [Zefau/ioBroker.jarvis#705]
Hi zusammen, habe gerade gesehen dass es für vis einen Slideshow Adapter gibt. Wäre das für Jarvis auch möglich?

**Comment by testit2020** [comment]
Kannst du Dir selber erstellen: https://forum.iobroker.net/topic/30668/jarvis-v2-2-0-just-another-remarkable-vis/3838

**Comment by ChrisWoehrle** [comment]
> Kannst du Dir selber erstellen: https://forum.iobroker.net/topic/30668/jarvis-v2-2-0-just-another-remarkable-vis/3838

Hm, ich kann da nichts erkennen. Eventl. kannst du mir da auf die Sprünge helfen?

**Comment by testit2020** [comment]
Unter Slideshow verstehst du nach einer bestimmten Zeit die einzelnen TABs nacheinander anzuzeigen?
Mit dem Eintrag unter dem Link kann man einen Tab anspringen. (Ändert sich auf allen verbundenen Geräten gleichzeitig).
Jetzt kann man mit steuere diesen Eintrag an die jarvis.0.addNotification senden und das für unterschiedliche Tabs mit eine Schleife entsprechend einstellen.

**Comment by ChrisWoehrle** [comment]
Nein, ich möchte keine Tabs anzeigen sondern Bilder nach einer gewissen Zeit der Ineffektivität. Wie ein Bildschirmschoner. Gibt's bspw. Bei Echo Show, Windows,...--Diese Nachricht wurde von meinem Android Mobiltelefon mit WEB.DE Mail gesendet.Am 04.04.21, 13:40 schrieb testit2020 ***@***.***>:

   
  Unter Slideshow verstehst du nach einer bestimmten Zeit die einzelnen TABs nacheinander anzuzeigen? Mit dem Eintrag unter dem Link kann man einen Tab anspringen. (Ändert sich auf allen verbundenen Geräten gleichzeitig). Jetzt kann man mit steuere diesen Eintrag an die jarvis.0.addNotification senden und das für unterschiedliche Tabs mit eine Schleife entsprechend einstellen.

**Comment by ChrisWoehrle** [comment]
@Zefau 
Ist eine Pro-Version gepalnt? Frage wegen des Labels "pro".

**Comment by Zefau** [comment]
Jo, für bestimme Funktionen.
